test/spec/modules/teadsBidAdapter_spec.js

```diff
@@ -1,13 +1,23 @@
 import {expect} from 'chai';
+// import prebid.js to make processQueue available and flush hooks (in particular, those used for cookies)
+import 'src/prebid.js';
 import {spec} from 'modules/teadsBidAdapter.js';
 import {newBidder} from 'src/adapters/bidderFactory.js';
+import {getStorageManager} from 'src/storageManager';
 
 const ENDPOINT = 'https://a.teads.tv/hb/bid-request';
 const AD_SCRIPT = '<script type="text/javascript" class="teads" async="true" src="https://a.teads.tv/hb/getAdSettings"></script>"';
 
 describe('teadsBidAdapter', () => {
   const adapter = newBidder(spec);
 
+  before(function () {
+    // Following the introduction of tests involving reading/writing cookies,
+    // this allows for running this spec as a single file with:
+    // `gulp test --file "test/spec/modules/teadsBidAdapter_spec.js"`.
+    window.$$PREBID_GLOBAL$$.processQueue();
```

Comment:
With the stubbing of storage, you can get rid of this I believe.

This type of code in test files can cause downstream tests to be very flakey.

We do not want to modify the window object and its contents if we do not have to in tests.